code/script.js

```diff
@@ -0,0 +1,93 @@
+//DOM selectors
+const main = document.getElementById("projects"); // main portion of the site, contains all projects
+const usr = document.getElementById("usr"); // section in header that holds username and profile pic
+
+//Global variables
+const usrName = 'amandatange';
+let forkedRepos = []; // will contain filtered list of repos that have been forked
+let projectRepos = []; // will contain filtered list of forked repos with names that start with "project"
+
+const API_REPOS = `https://api.github.com/users/${usrName}/repos`
+
+const options = {
+    method: 'GET',
+    headers: {
+        Authorization: `token ${API_TOKEN}`
+    }
+}
+
+const fetchData = () => {
+    fetch(API_REPOS, options)
+    .then((res) => res.json())
+    .then((data) => { 
+        forkedRepos = data.filter(repo => repo.fork);
+        projectRepos = forkedRepos.filter(repo => repo.name.startsWith('project'));
+
+        const profilePic = data[0].owner.avatar_url;
+        const LinkToProfile = data[0].owner.html_url;
+
+        // for the user presentation
+        usr.innerHTML = `
+            <img class='avatar_url_pic' src='${profilePic}'/>
+            <h1><a href='${LinkToProfile}'>@${usrName}</a></h1>
+        `;
+
+        //Send completed projects length to build chart
+        myChart(projectRepos.length)
+
+        //Call function to fetch pull requests
+        fetchPR()
+    })
+}
+
+const fetchPR = () => {
+    for (let i = 0; i < projectRepos.length; i++) {
+        const repo = projectRepos[i];
+        const API_PR = `https://api.github.com/repos/technigo/${repo.name}/pulls?per_page=100`
+
+        //  since not all projects have a PR I don't have access to the commits from the PR API for all of them 
+        //  so I chose to use the commits api from the first fetch
+        //  but I think this way the number of commits includes those made by the technigo team before I forked the project?
```

Comment:
I think you are right here since looking at this repo, you made 5(?) commits and in total there are nine.